src/s3ql/umount.py

```diff
@@ -32,6 +32,7 @@ def parse_args(args):
         Unmounts an S3QL file system. The command returns only after all data
         has been uploaded to the backend.'''))
 
+    parser.add_log('~/.s3ql/umount.log')
```

Comment:
Let's omit the parameter here, I think None/stdout is a better default than a logfile.

Reply:
You're right, that makes sense. Updated the PR accordingly.